ninja-core/src/main/java/ninja/RouterImpl.java

```diff
@@ -137,8 +137,10 @@ public String getReverseRoute(
             return finalUrl;
 
         }
-
-        return null;
+        else {
+        	  logger.info("Could not find any reverse route for the method " + controllerMethodName + " of the Controller class " +controllerClass.getSimpleName());
```

Comment:
2 comments here:

1) Make sure to use spaces (there seem to be tabs)
2) Please rewrite the logging like that: 
```
logger.info("Could not find any reverse route for the method {} of the Controller class {}", controllerMethodName , controllerClass.getSimpleName());
```
The "+" may have negative performance indicatoins: http://www.slf4j.org/faq.html#logging_performance

Thanks!